osmosis-1/osmosis-1.assetlist.json

```diff
@@ -859,6 +859,34 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/huahua.png"
       }
+    },
+    {
+      "description": "The native token of Vidulum",
+      "denom_units": [
+        {
+          "denom": "ibc/E7B35499CFBEB0FF5778127ABA4FB2C4B79A6B8D3D831D4379C4048C238796BD",
+          "exponent": 0,
+          "aliases": [
+            "uvdl"
+          ]
+        },
+        {
+          "denom": "vdl",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/E7B35499CFBEB0FF5778127ABA4FB2C4B79A6B8D3D831D4379C4048C238796BD",
+      "name": "Vidulum",
+      "display": "vdl",
+      "symbol": "VDL",
+      "ibc": {
+        "source_channel": "channel-0",
+        "dst_channel": "channel-124",
+        "source_denom": "vdl"
```

Comment:
I don't think this is correct.  Should be `uvdl` right?

Reply:
Yes just pushed fix